src/main/kotlin/org/sol4k/Connection.kt

```diff
@@ -160,8 +172,8 @@ class Connection @JvmOverloads constructor(
         )
     }
 
-    fun sendTransaction(transaction: Transaction): String {
-        val encodedTransaction = Base64.getEncoder().encodeToString(transaction.serialize())
+    fun sendTransaction(transactionBytes: ByteArray): String {
```

Comment:
change parameter to serialized tx, so both `Transaction` and `VersionedTransaction` can use this method

Reply:
Let's add two other overloaded methods, it would be more convenient for API users